neo/public/stylesheets/main.css

```diff
@@ -97,3 +99,19 @@ search-bar {
   padding-left: 0;
   padding-right: 0;
 }
+
+#logo_img{
+  z-index = 0;
+  position: absolute;
+  top:2%;
+  left:10%;
+  width : 15%;
+  height: 10%;
+
+}
+
+.btn.btn-info{
```

Comment:
I left it in there so when they add the logo it will be auto positioned to the spot. I can remove it if we don't need it.